client/homescreen/activity-panel/panels.js

```diff
@@ -27,6 +30,18 @@ export function getAllPanels( {
 			),
 			title: __( 'Orders', 'woocommerce-admin' ),
 		},
+		reviewsEnabled === 'yes' && {
+			className: 'woocommerce-homescreen-card',
+			id: 'reviews-panel',
+			count: countUnapprovedReviews,
+			initialOpen: false,
+			panel: (
+				<ReviewsPanel
+					hasUnapprovedReviews={ countUnapprovedReviews > 0 }
+				/>
+			),
+			title: __( 'Reviews to moderate', 'woocommerce-admin' ),
```

Comment:
It wouldn't have been obvious because I think it was buried in another issue discussion but the copy for this should now just be "Reviews"